FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2BottomDrawerActivity.kt

```diff
@@ -411,18 +413,22 @@ private fun CreateDrawerWithButtonOnPrimarySurfaceToInvokeIt(
     drawerContent: @Composable ((() -> Unit) -> Unit),
 ) {
     val scope = rememberCoroutineScope()
-
+    var initialState by remember { mutableStateOf(DrawerValue.Closed) }
```

Comment:
can we make initialState part of bottomDrawerState only?

Reply:
Bottom Drawer state is part of the bottom drawer api only, this variable is used to remember the state with which we must call the drawer api after recomposition. 
I am working on introducing a new drawer state within the drawer which, when given as the argument for the initial state, remembers the last state automatically. 
Will merge that after this

Reply:
yeah but instead of passing this value in BottomDrawer, we can pass this as iniital value in DrawerState?

Reply:
Implemented that in the next PR:
https://github.com/microsoft/fluentui-android/pull/702